forge-gui/res/cardsfolder/b/bamboozling_beeble.txt

```diff
@@ -4,7 +4,7 @@ Types:Creature Beeble
 PT:1/1
 K:Protection from Robots
 A:AB$ Effect | Cost$ 1 T | ValidTgts$ Player | ReplacementEffects$ RigRoll | RememberObjects$ Targeted | ExileOnMoved$ Battlefield | SpellDescription$ The next time target player would roll one or more dice this turn, instead they roll that many dice plus one and you choose one of those rolls to ignore.
-SVar:RigRoll:Event$ RollDice | ValidPlayer$ Player.IsRemembered | ReplaceWith$ PlusRoll | Description$ The next time target player would roll one or more dice this turn, instead they roll that many dice plus one and you choose one of those rolls to ignore.
+SVar:RigRoll:Event$ RollDice | ActiveZones$ Command | ValidPlayer$ Player.IsRemembered | ReplaceWith$ PlusRoll | Description$ The next time target player would roll one or more dice this turn, instead they roll that many dice plus one and you choose one of those rolls to ignore.
```

Comment:
ActiveZone Command is already set by EffectEffect:

https://github.com/Card-Forge/forge/blob/master/forge-game/src/main/java/forge/game/ability/effects/EffectEffect.java#L218